doc/content/source/userobjects/ThresholdHeuristicUserObject.md

```diff
@@ -0,0 +1,24 @@
+# ThresholdHeuristicUserObject
+
+## Description
+
+`ThresholdHeuristicUserObject` inherits from the `ClusteringHeuristicUserObjectBase`. It implements the clustering
+heuristic if their scores are more (default) or less than a `threshold`
+value provided in the input files. The operation will be flipped if the 
+input argument `cluster_if_above_threshold` is false. 
+
+!alert! warning
+
+`ThresholdHeuristicUserObject` doesn't implement the clustering and mesh walking process.
```

Comment:
Is this warning necessary to have in the documentation? Or, asked another way, is there something in the source code which will throw an error if this user object were added but the BooleanCombo one was missing?  As much as possible, we should rely on coded error/warning messages and not documentation, since the code-based messages are guaranteed to catch errors whereas not everyone reads documentation thoroughly.